.eslintrc

```diff
@@ -9,7 +9,6 @@
     "arrow-spacing": [2],
     "babel/arrow-parens": [2, "as-needed"],
     "brace-style": [2, "1tbs", {"allowSingleLine": true}],
-    "camelcase": [2, {"properties": "always"}],
```

Comment:
I added this recently but it is really silly, especially when working with data from an API I don't control. Let's remove it and be less strict, yay!